website/docs/d/odb_db_servers.html.markdown

```diff
@@ -46,7 +46,7 @@ This data source exports the following attributes in addition to the arguments a
 * `cpu_core_count` - The number of CPU cores enabled on the database server.
 * `created_at` - The date and time when the database server was created.
 * `db_node_storage_size_in_gbs` - The amount of local node storage, in gigabytes (GB), that's allocated on the database server.
-* `db_server_id` - The unique identifier of the database server.
+* `db_servers[*].id` - The unique identifiers of the database servers.
```

Comment:
Since this list of attributes documents what is contained in the `db_servers` attribute, I think this should just be `id`.


```suggestion
* `id` - The unique identifier of the database server.
```